### ISSUE TITLE
Image gallery icon and RSS feed icon on now.uiowa.edu should have alt text or be marked as decorative

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#9429]
The alt text checker from https://www.w3.org/WAI/test-evaluate/easy-checks/image-alt/ is flagging the image gallery icon on https://now.uiowa.edu/news/2025/05/ui-professor-author-melissa-febos-talks-about-books-she-keeps-and-why and the RSS feed icon on pages like https://now.uiowa.edu/tags/featured-headlines, https://now.uiowa.edu/research-areas/virtual-reality as missing alt text/not marked as decorative. These are not being flagged by Siteimprove. 

# Solution
Update the icons to have alt text or be marked as decorative.
